modules/mesh-layers/src/utils/matrix.js

```diff
@@ -0,0 +1,125 @@
+import {Matrix4} from 'math.gl';
+
+const RADIAN_PER_DEGREE = Math.PI / 180;
+
+export const matrixLayerProps = {
```

Comment:
matrixLayerProps is kind of confusing, looks like we have a MatrixLayer somewhere.

Reply:
Fixed according to @ibgreen comments.